server/src/controllers/userController.js

```diff
@@ -52,3 +57,6 @@ exports.login = async function (req, res) {
     res.status(400).json({ message: "Please fill the required fields" });
```

Comment:
On the front-end, you're verifying that the user is logged in using the `userData.message` string, yeah? I think it's better to do that verification with something other than the message, since you're sending a message even when there's an error in the backend.